substratest/client.py

```diff
@@ -85,9 +85,34 @@ def add_testtuple(self, spec):
 
     def add_compute_plan(self, spec):
         res = self._client.add_compute_plan(spec.to_dict())
+        # The API outputs of the create and get/list methods differ at the
+        # moment. This harmonizes the returns so that they can be parsed.
+        # See:
+        # https://github.com/SubstraFoundation/substra-chaincode/issues/36
+        res['traintuples'] = res['traintupleKeys']
+        res['testtuples'] = res['testtupleKeys']
+        del res['traintupleKeys']
+        del res['testtupleKeys']
+        res['algoKey'] = spec.algo_key
+        res['objectiveKey'] = spec.objective_key
```

Comment:
I would prefer to create an `assets.ComputePlanCreated` structure instead. 

This will be also consistent with how we handle DataSample (different responses when doing read or write operations).

Reply:
On it!